aten/src/ATen/native/native_functions.yaml

```diff
@@ -2941,6 +2941,10 @@
 
 - func: max_pool1d(Tensor self, int[1] kernel_size, int[1] stride=[], int[1] padding=0, int[1] dilation=1, bool ceil_mode=False) -> Tensor
 
+- func: _max_pool1d_forward(Tensor self, int[1] kernel_size, int[1] stride=[], int[1] padding=0, int[1] dilation=1, bool ceil_mode=False) -> Tensor
+  dispatch:
+    CompositeExplicitAutograd: _max_pool1d_forward
```

Comment:
Because this is CPU-only, can we do "dispatch: -cpu: _max_pool1d_forward"?

Reply:
Have updated the dispatch to CPU-only and also updated the name to `_max_pool1d_cpu_forward`.